.github/workflows/deploy.yml

```diff
@@ -58,7 +58,8 @@ jobs:
         env:
           APP_BUCKET: ${{ secrets.APP_BUCKET }}
           AWS_REGION: ${{ secrets.AWS_REGION }}
-          DOCKERHUB_REPOSITORY: ${{ secrets.DOCKERHUB_REPOSITORY }}
+          SERVER_DOCKERHUB_REPOSITORY: ${{ secrets.SERVER_DOCKERHUB_REPOSITORY }}
```

Comment:
🤔 

I understand the push for clarity, but slight preference to keep the original `DOCKERHUB_REPOSITORY` name as-is to reduce regression risk.